doc/modules/clustering.rst

```diff
@@ -1582,3 +1582,36 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format describing the relationship between
+discrete-valued random variables.
+It shows the multivariate frequency distribution of the variables.
+
+Let the first random variable be :math:`X`, and the second random
+variable be :math:`Y`.
+In a contingency matrix :math:`M`, entry :math:`M(i, j)` is the number of
+observations with :math:`X=X_i` and :math:`Y=Y_j`.
+
+Here is an example:
```

Comment:
Here is an example: -> Here is an example::

Reply:
Does this make any visual difference in .rst files? May be this is used for code?

Reply:
:: is for code block in RST and sphinx. This is the usual way of writing doc.

Reply:
you can check the section "Source code": http://www.sphinx-doc.org/en/stable/rest.html